src/View/SSViewer_FromString.php

```diff
@@ -3,9 +3,11 @@
 namespace SilverStripe\View;
 
 use SilverStripe\Core\Config\Config;
+use SilverStripe\Dev\Deprecation;
 
 /**
  * Special SSViewer that will process a template passed as a string, rather than a filename.
+ * @deprecated 5.4.0 Will be replaced with SilverStripe\View\SSTemplateEngine::renderString()
  */
 class SSViewer_FromString extends SSViewer
```

Comment:
Doesn't make sense to have this abstraction anymore - if you have a template string, it'll be for a specific template syntax so you should make sure to get the correct template engine from the injector and ask it to render your template string directly.